### PR TITLE
[AIG] Add Evaluations section with Overview and How to set up Evaluations p…

### PR DESCRIPTION
**Pull request by daisyfaithauma**
- Added Evaluations page
- Added overview and how to set up

### CODE REVIEW

**Review by kathayl — APPROVED** [review]

File: src/content/docs/ai-gateway/observability/evaluations/index.mdx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Evaluations
+pcx_content_type: navigation
+order: 1
+---
+
+# Overview
```
Comment (Oxyjun) [inline src/content/docs/ai-gateway/observability/evaluations/index.mdx]:
```suggestion
## Overview
```

I've personally never seen a H1. Also, do we need this part at all? I've also never seen a file that starts with a header (usually it's title, then text, then first H2)

File: src/content/docs/ai-gateway/observability/evaluations/index.mdx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Understanding your application's performance is essential for optimization. Developers often have different priorities, and finding the optimal solution involves balancing key factors such as cost, latency, and accuracy. Some prioritize low-latency responses, while others focus on accuracy or cost-efficiency.
+
+AI Gateway's Evaluations provide the data needed to make informed decisions on how to optimize your AI application. Whether it's adjusting the model, provider, or prompt, this feature delivers insights into key metrics around performance, speed, and cost. It empowers developers to better understand their application's behavior, ensuring improved accuracy, reliability, and customer satisfaction.
+
+Evaluations use datasets which  are collections of logs stored for analysis. You can create datasets by applying filters in the Logstab, which help narrow down specific logs for evaluation.
```
Comment (Oxyjun) [inline src/content/docs/ai-gateway/observability/evaluations/index.mdx]:
```suggestion
Evaluations use datasets which are collections of logs stored for analysis. You can create datasets by applying filters in the Logs tab, which help narrow down specific logs for evaluation.
```

File: src/content/docs/ai-gateway/observability/evaluations/set-up-evaluations.mdx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+:::
+
+## 2. Select evaluators
+
+After creating a dataset, choose the evaluation parameters:
+
+- Cost: Calculates the average cost of inference requests within the dataset (only for requests with [cost data](/ai-gateway/observability/costs/)).
+- Speed: Calculates the average duration of inference requests within the dataset.
+- Performance:
+  - Human feedback: Measures performance based on human feedback, calculated by the % of thumbs up on the logs, annotated from the Logs tab.
```
Comment (Oxyjun) [inline src/content/docs/ai-gateway/observability/evaluations/set-up-evaluations.mdx]:
```suggestion
  - Human feedback: measures performance based on human feedback, calculated by the % of thumbs up on the logs, annotated from the Logs tab.
```

Using lowercase for consistency with above bullets, feel free to ignore if deliberate

File: src/content/docs/ai-gateway/observability/evaluations/index.mdx (hunk 1 above)
Comment (daisyfaithauma) [reply src/content/docs/ai-gateway/observability/evaluations/index.mdx]: My bad.

File: src/content/docs/ai-gateway/observability/evaluations/set-up-evaluations.mdx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+Additional evaluators will be introduced in future updates to expand performance analysis capabilities.
+
+:::
+
+## 3. Name, review, and run the evaluation
+
+1. Create a unique name for your evaluation to reference it in the dashboard.
+2. Review the selected dataset and evaluators.
+3. Select **Run** evaluation to start the process.
```
Comment (Oxyjun) [inline src/content/docs/ai-gateway/observability/evaluations/set-up-evaluations.mdx]:
```suggestion
3. Select **Run** to start the process.
```
Should this be "**Run evaluation** or just **Run**?

Comment (daisyfaithauma) [reply src/content/docs/ai-gateway/observability/evaluations/set-up-evaluations.mdx]: Just Run actually.

**Review by Oxyjun — APPROVED** [review]

**Comment by daisyfaithauma** [comment]
Could not build. Redone in another PR

**Pull request closed without merging**